doc/api/buffer.markdown

```diff
@@ -251,9 +251,9 @@ console.log(buf2.toString());
 
 ### Class Method: Buffer.byteLength(string[, encoding])
 
-* `string` String
-* `encoding` String, Optional, Default: 'utf8'
-* Return: Number
+* `string` {String}
+* `encoding` {String} Default: `'utf8'`
+* Return: {Number}
 
 Returns the actual byte length of a string. If not specified, `encoding`
 defaults to `'utf8'`. This is not the same as [`String.prototype.length`][]
```

Comment:
@silverwind, good point. Do you have any opinions on the [issue above](https://github.com/nodejs/node/pull/4873#discussion_r50840836)?